LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1095,9 +1095,16 @@ function getForm(indicatorID, series) {
                     </tr>\
                     <tr>\
                         <td>Disabled</td>\
-                        <td colspan="1"><input id="disabled" name="disabled" type="checkbox" /></td>\
+                        <td colspan="1"><input id="disabled" name="disable_or_delete" type="radio" /></td>\
```

Comment:
My only concern for these buttons user wise is if they accidently click it they can't unclick either or because its a radio, so one always has to be selected. I would like to see it as a grouped checkbox system, only one of them can be checked at a time or make it disable the other checkbox if one is selected so users can unselect if they change their mind and made changes.

Reply:
This might help: http://wastedpotential.com/html-multi-checkbox-set-the-correct-way-to-group-checkboxes/